packages/connect-query-core/src/create-infinite-query-options.ts

```diff
@@ -117,7 +116,6 @@ export function createInfiniteQueryOptions<
     | SkipToken;
   structuralSharing: (oldData: unknown, newData: unknown) => unknown;
   initialPageParam: MessageInitShape<I>[ParamKey];
-  queryKeyHashFn: (queryKey: QueryKey) => string;
```

Comment:
Can we remove this in a separate PR? We can't take it back after publishing v2.

Reply:
Filed [here](https://github.com/connectrpc/connect-query-es/pull/470)